pkg/services/accesscontrol/resolvers.go

```diff
@@ -44,7 +44,7 @@ type Resolvers struct {
 }
 
 func (s *Resolvers) AddScopeAttributeResolver(prefix string, resolver ScopeAttributeResolver) {
-	s.log.Debug("adding scope attribute resolver for '%v'", prefix)
+	s.log.Debug(fmt.Sprintf("adding scope attribute resolver for '%v'", prefix))
```

Comment:
Nit. Avoid parameterization in favor of key-value pairs for additional data https://github.com/grafana/grafana/blob/main/contribute/engineering/backend/instrumentation.md#naming-conventions